src/py/_pyodide/_core_docs.py

```diff
@@ -441,7 +448,35 @@ def reverse(self) -> None:
         """
 
 
+@Mapping.register
 class JsMap(JsProxy):
+    """A JavaScript Map
+
+    To be considered a map, a JavaScript object must have a ``.get`` method, it
+    must have a ``.size`` or a ``.length`` property which is a number
+    (idiomatically it should be called ``.size``) and it must be iterable.
+    """
+
+    _js_type_flags = ["HAS_GET | HAS_LENGTH | IS_ITERABLE", "IS_OBJECT_MAP"]
+
+    def __getitem__(self, idx: Any) -> Any:
+        return None
+
+    def __setitem__(self, idx: Any, value: Any) -> None:
+        pass
+
+    def __delitem__(self, idx: Any) -> None:
```

Comment:
Isn't `__setitem__` and `__delitem__` for `JsMutableMap`?

Reply:
Yes good point.